packages/extension/src/pages/register/connect-keystone/scan.tsx

```diff
@@ -0,0 +1,136 @@
+import { observer } from "mobx-react-lite";
+import React, { FunctionComponent, useState } from "react";
+import { useRegisterHeader } from "../components/header";
+import {
+  useSceneEvents,
+  useSceneTransition,
+} from "../../../components/transition";
+import { RegisterSceneBox } from "../components/register-scene-box";
+import KeystoneSDK, { UR } from "@keystonehq/keystone-sdk";
+import { Box } from "../../../components/box";
+import { ColorPalette } from "../../../styles";
+import { CameraIcon } from "../../../components/icon";
+import { Purpose, useAnimatedQRScanner } from "@keystonehq/animated-qr";
+import { GuideBox } from "../../../components/guide-box";
+import { KeystoneError } from "../../../components/keystone/error";
+
+export const ScanKeystoneScene: FunctionComponent<{
+  name: string;
+  password: string;
+
+  // append mode일 경우 위의 name, password는 안쓰인다. 대충 빈 문자열 넣으면 된다.
+  appendModeInfo?: {
+    vaultId: string;
+    afterEnableChains: string[];
+  };
```

Comment:
@renfengshi I think I'll delete the `appendModeInfo` parameter since it's not used in the current file.